src/js-display-panel.ts

```diff
@@ -0,0 +1,135 @@
+import { ToggleStatus } from './js-feature-toggle-manager';
+import { QueryStringReader } from './github/gtmsportswear/js-utilities@1.0.0/js-utilities';
+
+export abstract class DisplayPanel {
```

Comment:
I used an abstract class for `DisplayPanel` assuming that we are going to have multiple possible display panels used once the idea of A/B testing gets into play. This would allow us to create other types of display panels that take different types of data without having to affect this one.